R/regularization.R

```diff
@@ -1,3 +1,59 @@
+#' Regularization via L1 norm
```

Comment:
Maybe more specific docs? This IS going to come back at us at some point ;)

Reply:
https://github.com/kkmann/otsd/blob/05a05ebe3e04965acf67afae79621af6c2b4e917/R/regularization.R#L3-L9